packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -352,7 +356,8 @@ export const getActivitiesByAddress = async function (
         return null;
```

Comment:
Two lines above, direct `console.warn` usage in production code should be replaced with configurable logging.

Recommended fix:
```typescript
if (process.env.NODE_ENV === 'development') {
  console.warn(`Skipping transaction ${transactionID} due to: ${(err as Error).message}`);
}
```

Reply:
Or do we want to log this anyway?